Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8-alpine as dependencies
+WORKDIR /usr/src/app/
+RUN yarn add renovate --production --no-lockfile
```

Comment:
I'd prefer instead to add the source code (e.g. `lib` so that it will be in-sync with the repo)